src/Sulu/Component/Localization/Localization.php

```diff
@@ -69,6 +69,12 @@ class Localization implements \JsonSerializable, ArrayableInterface
      */
     private $xDefault;
 
+    public function __construct($language = null, $country = null)
+    {
+        $this->language = $language;
+        $this->country = $country;
+    }
```

Comment:
Actually it would be an even better idea to make this class immutable, but sadly we can't do that... I'll create an issue for that.